src/errors/helper/assertionErrorHelper.js

```diff
@@ -1,5 +1,6 @@
 const REQUIRED_PARAMENTER_MISSING = 'REQUIRED_PARAMENTER_MISSING';
 const REQUIRED_PARAMETER_TO_BE_NON_EMPTY_STRING = 'REQUIRED_PARAMETER_TO_BE_NON_EMPTY_STRING';
+const REQUIRED_PARAM_SHOULD_MATCH_EMAIL_REGEX = 'REQUIRED_PARAM_SHOULD_MATCH_EMAIL_REGEX';
```

Comment:
Can we please remove the helpers folder and move the file to top. 
src/errors/assertionHelper.js